src/Datadog.Trace/Tags.cs

```diff
@@ -12,6 +12,16 @@ public static class Tags
         /// </summary>
         public const string Env = "env";
 
+        /// <summary>
+        /// The version of the profiled service.
+        /// </summary>
+        public const string Version = "version";
+
+        /// <summary>
+        /// The version of the profiled service, specified in the OpenTracing style.
+        /// </summary>
+        public const string OpenTracingVersion = "service.version";
```

Comment:
@jdgumz has this been agreed upon consistently across APM integrations?

Reply:
The constant above where `Version="version"` is the tag we primarily work with. This is for convenience, so when the `service.version` tag is set, we also set `version` tag, which is the main, standardized property

Reply:
Yes this is a specific detail for folks using OpenTracin: if `service.version` is present then we "duplicate" its value into a separate tag called `version`.

I'm fairly sure this is the idea we've implemented elsewhere.

Reply:
👍 yes, we support people manually setting via `service.version`, which should result in both `version` and `service.version` being set.

Both tags will be present because if someone sets `service.version`... `service.version` should be there.

Reply:
I'm going to resolve this conversation but let me know if it should be re-opened